try {
    throw "oh no!";
} catch (e) {
    console.log("Oh well.");
}
finally {
    console.log("oh my");
}